### ISSUE TITLE
Documenting content team workflow process

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#4426]
## User Story or Problem Statement
As a member of the Public Websites content team, I need to understand the workflow process of pages from creation to publication so I can better do my job.

## Goal
Create a document that all content team members can refer to when working on new or revised pages for VA.gov. This will be a step-by-step guide of how to take a Tier 1 page from concept to publication. 

## Objectives or Key Results this is meant to further
An organized workflow for the content team to follow

## Acceptance Criteria
- [x] @RLHecht creates a workflow process list based on past practice and previous documentation
- [x] @AnneHurley and @peggygannon review and add additional information
- [ ] @jenniferlee-dsva and @Karak888 do final review

## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `triage`, `tools-improvements`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `call center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by RLHecht** [comment]
@peggygannon @AnneHurley Here is the Word doc for the copy flow process. Can you both take a look and let me know what's missing or unclear? I also highlighted some questions I didn't know the answers to. Thanks![https://app.zenhub.com/files/133843125/82beee36-639e-4d52-93ae-f4766ecfd25d/download](https://app.zenhub.com/files/133843125/82beee36-639e-4d52-93ae-f4766ecfd25d/download)

**Comment by AnneHurley** [comment]
New step 
***Depending on the extent of the revisions, the writer may need to create copy decks for the SMEs/stakeholders to review. See sample template attached. Be sure to include:

- - The correct naming convention (see example)
- - The official Veterans Affairs logo
- - Introductory copy that sets up the why and how of “plain language” and specific questions/background if applicable
- - Links to legacy and source pages used when creating the copy deck

Take screen shots of the pages built in GitHub (Heroku preview) or Drupal Prod/Preview, and paste in order to make the complete web page. Draw red boxes around any new content you want the SMEs to review, so they can find the updates quickly. They should not need to review anything outside the red-boxed updates.

Turn the copy deck into a PDF and then the SMEs can make their edits in the PDF stickies.

Example: [Brand Consolidation Updates Burials and Memorials Deck 1 of 4 Example]
(https://app.zenhub.com/files/133843125/64f8ac21-fe6f-4eeb-8341-bf4ec0647ec6/download)

**Comment by RLHecht** [comment]
Attached is the new Word doc with Anne's additional step on copy decks. @peggygannon can you also please review and let me know if there are any steps that are missing or unclear? There are some questions I still have, which I highlighted in the doc.
[Workflow Process Doc.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/4013139/Workflow.Process.Doc.docx)

**Comment by peggygannon** [comment]
@RLHecht @AnneHurley Hello, here is the workflow process doc with some feedback from me. I'm also uploading the former website team content process document created in 2018. You might want to integrate some steps or formatting from that document into your new one? 

[Workflow.Process.pgedits.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/4017533/Workflow.Process.pgedits.docx)

*Former content process doc from 2018*
[Content Team Process and Tools.5.6.18.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/4017537/Content.Team.Process.and.Tools.5.6.18.docx)

**Comment by RLHecht** [comment]
@peggygannon and @AnneHurley Here is my edited document with Peggy's changes from last week. I still have a few questions highlighted in yellow. Also, Peggy you can add your review steps in as well. Thanks![https://app.zenhub.com/files/133843125/47c1365c-9a03-4e83-93a3-d29041ade767/download](https://app.zenhub.com/files/133843125/47c1365c-9a03-4e83-93a3-d29041ade767/download)

**Comment by peggygannon** [comment]
Hi, @RLHecht I'm not able to open this document. Can you try uploading it again?

**Comment by RLHecht** [comment]
@peggygannon I just reuploaded it in the comment above. Let me know if it still doesn't work. Sorry about that!

**Comment by peggygannon** [comment]
@RLHecht Hi, Randi, here is the document with additional updates from me. Please let me know if you have any questions or want to discuss over the phone. 

[Workflow.Process.Doc.1.pgedits.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/4042078/Workflow.Process.Doc.1.pgedits.docx)

**Comment by RLHecht** [comment]
Thanks, @peggygannon ! I think all these edits make sense to me. @AnneHurley Can you look at Peggy's version in the comment above? There's a question or two in there you might be able to answer. Also please read through it and let me know if the process makes sense and does not sound too confusing. I want it to be thorough but also not hard to follow. Then I can send to Kara/Jen for them to review. @Karak888

**Comment by RLHecht** [comment]
@AnneHurley Can you please take a look at the latest document Peggy edited and let me know if you have more changes as well as answer the few questions that are still in there? Thanks!

**Comment by aeltarhoni** [comment]
@Karak888 will review to see if this provides the solution she needs. If so, it can be Closed and a new ticket will be created on the content ticketing process.
